### DISCUSSION TITLE
Model Conversion Progress indicator

### DISCUSSION THREAD

**Discussion by AlyceOsbourne** [nomic-ai/pygpt4all#32] [Ideas]
(EDIT: OK, it just takes a long time to actually start the process, there is logging there after)

Is there a chance that a progress indicator could be added to the converter?
In the version included in utils, you could wrap the loop in a TQDM bar, I am unsure of where the cli version is to make recommendations there

**Comment by absadiki** [comment]
@AlyceOsbourne, Yeah there is some logging, 
it should not take a lot of time anyway.
